config/prod/s3-synapse-sync.yaml

```diff
@@ -8,6 +8,7 @@ stack_tags:
 dependencies:
   - "prod/htan-synapse-sync-kms-key.yaml"
 parameters:
+  CIDR: "10.50.0.0/16"
```

Comment:
`10.50.0.0/16` is union station right? does this need to be `10.100.0.0/16` for vpn client access? i'm not entirely sure which to use